src/Map/SpatialId/utils.ts

```diff
@@ -77,3 +49,16 @@ export const getSpaceData = (space: Space): SpatialIdSpaceData => {
     vertices: space.vertices3d(),
   };
 };
+
+export const getVerticalLimits = (
+  maxHeigth: number,
+  minHeight: number,
+  zoom: number,
+): { top: number; bottom: number } => {
+  // lat/lng doesn't matter
+  const topSpace = new Space({ lat: 35, lng: 139, alt: maxHeigth }, zoom);
+  const bottomSpace = new Space({ lat: 35, lng: 139, alt: minHeight }, zoom);
```

Comment:
Personally, 0 would be better
```suggestion
  // lat/lng doesn't matter
  const topSpace = new Space({ lat: 0, lng: 0, alt: maxHeigth }, zoom);
  const bottomSpace = new Space({ lat: 0, lng: 0, alt: minHeight }, zoom);
```